src/gui/mzroll/mainwindow.cpp

```diff
@@ -1587,6 +1587,17 @@ void MainWindow::analyticsAverageSpectra(){
 
 void MainWindow::open()
 {
+    // TODO: temporarily added for informing user, remove after a few releases
+    if (!_versionRecordExists()) {
+        QMessageBox msgBox;
+        msgBox.setText("El-MAVEN is now capable of reading files containing "
```

Comment:
@surbhipoddar Here's the message we're displaying on the File loading page. Let us know if this is fine or if any modification is required.